client/modules/client_state_machine.js

```diff
@@ -208,6 +208,18 @@ define(function(require) {
     constructor() {
       // Initially, we tell the clients to show a blank screen.
       super(new DisplayState(ClientModule.newEmptyModule()), debug);
+      
+      this.setErrorListener(error => {
+        if (monitor.isEnabled()) {
+          monitor.update({client: {
+            event: error.toString(),
+            time: timeManager.now(),
+            color: [255, 0, 0]
```

Comment:
Color?

Reply:
In the monitor dashboard (&monitor=1 in the client, --enable_monitoring in the server), we show this message in red because it's an ERROR.

Reply:
type: 'error'?

Picking the color seems overly specific...  /shrug

Reply:
That's reasonable. I'll file a bug to improve the monitor API to specify this information there, rather than 'color'.

Reply:
#168